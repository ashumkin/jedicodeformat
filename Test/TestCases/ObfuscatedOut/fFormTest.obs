unit fformtest;interface uses windows,messages,sysutils,classes,graphics,controls
,forms,dialogs,stdctrls,buttons;type tformtest=class(TForm)bbclose:tbitbtn;mnotice
:tmemo;procedure bbcloseclick(sender:tobject);private public end;var formtest:tformtest
;implementation {$R *.DFM}procedure tformtest.bbcloseclick(sender:tobject);begin 
close;application.terminate;halt;end;end.