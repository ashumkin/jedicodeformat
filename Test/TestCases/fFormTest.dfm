object FormTest: TFormTest
  Left = 192
  Top = 107
  BorderStyle = bsDialog
  Caption = 'JCF Test Code Project'
  ClientHeight = 270
  ClientWidth = 542
  Color = clBtnFace
  Font.Charset = ANSI_CHARSET
  Font.Color = clWindowText
  Font.Height = -15
  Font.Name = 'Segoe UI'
  Font.Style = []
  OldCreateOrder = False
  Scaled = False
  PixelsPerInch = 120
  TextHeight = 20
  object bbClose: TBitBtn
    Left = 214
    Top = 217
    Width = 123
    Height = 49
    Caption = '&Close'
    TabOrder = 0
    OnClick = bbCloseClick
    Glyph.Data = {
      DE010000424DDE01000000000000760000002800000024000000120000000100
      0400000000006801000000000000000000001000000000000000000000000000
      80000080000000808000800000008000800080800000C0C0C000808080000000
      FF0000FF000000FFFF00FF000000FF00FF00FFFF0000FFFFFF00388888888877
      F7F787F8888888888333333F00004444400888FFF444448888888888F333FF8F
      000033334D5007FFF4333388888888883338888F0000333345D50FFFF4333333
      338F888F3338F33F000033334D5D0FFFF43333333388788F3338F33F00003333
      45D50FEFE4333333338F878F3338F33F000033334D5D0FFFF43333333388788F
      3338F33F0000333345D50FEFE4333333338F878F3338F33F000033334D5D0FFF
      F43333333388788F3338F33F0000333345D50FEFE4333333338F878F3338F33F
      000033334D5D0EFEF43333333388788F3338F33F0000333345D50FEFE4333333
      338F878F3338F33F000033334D5D0EFEF43333333388788F3338F33F00003333
      4444444444333333338F8F8FFFF8F33F00003333333333333333333333888888
      8888333F00003333330000003333333333333FFFFFF3333F00003333330AAAA0
      333333333333888888F3333F00003333330000003333333333338FFFF8F3333F
      0000}
    NumGlyphs = 2
  end
  object mNotice: TMemo
    Left = 5
    Top = 5
    Width = 528
    Height = 208
    Font.Charset = ANSI_CHARSET
    Font.Color = clWindowText
    Font.Height = -20
    Font.Name = 'Calibri'
    Font.Style = [fsBold]
    Lines.Strings = (
      'This is the test code for JEDI Code Format (JCF).'
      'There should be no need to ever run this project.'
      'It exists solely to verify that the test code is '
      'compilable.'
      ''
      'Go to View|Project Manager')
    ParentColor = True
    ParentFont = False
    ReadOnly = True
    TabOrder = 1
  end
end
