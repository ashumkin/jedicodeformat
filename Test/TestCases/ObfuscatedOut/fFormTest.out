unit fformtest;

interface

uses Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs, StdCtrls, Buttons;

type
  tformtest = class(TForm)
    bbclose: tbitbtn;
    mnotice: tmemo;
    procedure bbcloseclick(Sender: TObject);
  private
  public
  end;

var
  formtest: tformtest;

implementation

{$R *.DFM}

procedure tformtest.bbcloseclick(Sender: TObject);
begin
  Close;
  application.terminate;
  halt;
end;

end.
